data/vanderMoezel_1987/metadata.yml

```diff
@@ -115,9 +115,9 @@ traits:
     species was seen flowering in a four-year-old site, was not seen flowering in
     a two-year-old site and was not recorded in a three-year-old site, so therefore
     it flowers within three or four years after a fire.
-- var_in: reproductive_maturity_secondary_after_fire
+- var_in: juvenile_period
   unit_in: a
-  trait_name: reproductive_maturity_secondary_after_fire
+  trait_name: fire_time_from_fire_to_flowering
```

Comment:
Doesn't an entity context for 'time to flowering type' need to be added for this?

Reply:
yes, I've done that